docs/root/intro/version_history.rst

```diff
@@ -13,6 +13,8 @@ Version history
   `google.api.HttpBody <https://github.com/googleapis/googleapis/blob/master/google/api/httpbody.proto>`_.
 * cluster: added :ref:`option <envoy_api_field_Cluster.CommonLbConfig.update_merge_window>` to merge
   health check/weight/metadata updates within the given duration.
+* cluster: added the ability to completely disable circuit breakers in the
+  absence of its :ref:`configuration <envoy_api_field_Cluster.circuit_breakers>`.
```

Comment:
You need to add to circuit breaker docs as well. So that uses know that absence of circuit breaker means circuit breaker is disabled

Reply:
Addressed in 8dc1e02.